release-notes/opensearch-release-notes-2.19.3.md

```diff
@@ -0,0 +1,219 @@
+
+
+* Validate request fields in DDB Put and Update implementations ([#157](https://github.com/opensearch-project/opensearch-remote-metadata-sdk/pull/157))
+* Properly handle remote client search failures with status codes ([#158](https://github.com/opensearch-project/opensearch-remote-metadata-sdk/pull/158))
+
+
+### SQL
+
+
+* Fix: Date field format parsing for legacy query engine ([#3160](https://github.com/opensearch-project/sql/pull/3160))
+
+
+## INFRASTRUCTURE
+
+
+### SQL
+
+
+* Increment version to 2.19.3-SNAPSHOT ([#3601](https://github.com/opensearch-project/sql/pull/3601))
+
+
+## DOCUMENTATION
+
+
+### Opensearch Alerting
+
+
+* Added 2.19.3.0 release notes. ([#1897](https://github.com/opensearch-project/common-utils/pull/1897))
+
+
+### Opensearch Alerting Dashboards Plugin
+
+
+* Added 2.19.3 release notes. ([#1269](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/1269))
+
+
+### Opensearch Common Utils
+
+
+* Added 2.19.3.0 release notes. ([#854](https://github.com/opensearch-project/common-utils/pull/854))
+
+
+### Opensearch Query Insights
+
+
+* 2.19.3 Release Notes ([#386](https://github.com/opensearch-project/query-insights/pull/386))
+
+
+### Opensearch Query Insights Dashboards
+
+
+* 2.19.3 Release Notes ([#280](https://github.com/opensearch-project/query-insights-dashboards/pull/280))
+
+
+## MAINTENANCE
+
+
+### Opensearch Alerting
+
+
+* Pinned the commons-beanutils dependency to 1.11.0 version ([#1887](https://github.com/opensearch-project/common-utils/pull/1887))
+* [2.19] Moved the commons-beanutils pinning to the core gradle file ([#1893](https://github.com/opensearch-project/common-utils/pull/1893))
+
+
+### Opensearch Alerting Dashboards Plugin
+
+
+* Fix CVE ([#1268](https://github.com/opensearch-project/alerting-dashboards-plugin/pull/1268))
+
+
+### Opensearch Anomaly Detection
+
+
+* Migrating from commons-lang2.6 to commons-lang3.18 ([#1526](https://github.com/opensearch-project/anomaly-detection/pull/1526))
+
+
+### Opensearch Anomaly Detection Dashboards
+
+
+* Bump elliptic to 6.6.1 ([#1062](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/1062))
+
+
+### Opensearch Common Utils
+
+
+* Increment version to 2.19.3-SNAPSHOT ([#826](https://github.com/opensearch-project/common-utils/pull/826))
+* Pinned the commons-beanutils dependency to fix CVE-2025-48734 ([#850](https://github.com/opensearch-project/common-utils/pull/850))
+
+
+## Bug fix
+
+
+* validate that index patterns are not allowed in create/update doc level monitor ([#829](https://github.com/opensearch-project/common-utils/pull/829))
+* Fix isDocLevelMonitor check to account for threat intel monitor ([#835](https://github.com/opensearch-project/common-utils/pull/835))
+* updating PublishFindingsRequest to use a list of findings rather than... ([#832](https://github.com/opensearch-project/common-utils/pull/832))
```

Comment:
```suggestion
* Updating PublishFindingsRequest to use a list of findings rather than... ([#832](https://github.com/opensearch-project/common-utils/pull/832))
```